packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/notebook-resources.html

```diff
@@ -0,0 +1,152 @@
+    <div>
+        <!--Header-->
+        <p class="oui-heading_5 mb-2">
+            <strong
+                data-translate="pci_notebook_add_resources_usecase"
+            ></strong>
+        </p>
+
+        <!--Use cases-->
+        <div class="row">
+            <oui-select-picker
+                class="d-inline-block col-md-6 col-xl-3 my-3"
+                data-name="notebookResources"
+                data-label="{{:: 'pci_notebook_add_resources_usecase_item_standard' | translate}}"
+                data-description="{{:: 'pci_notebook_add_resources_usecase_item_standard_description' | translate}}"
+                data-model="$ctrl.notebook.selected.resource.usage"
+                data-values="[$ctrl.NOTEBOOK_RESOURCES.STANDARD]"
+                data-on-change="$ctrl.onUsecaseChange($ctrl.NOTEBOOK_RESOURCES.STANDARD_FLAVOR)"
+            >
+            </oui-select-picker>
+
+            <oui-select-picker
+                class="d-inline-block col-md-6 col-xl-3 my-3"
+                data-name="notebookResources"
+                data-label="{{:: 'pci_notebook_add_resources_usecase_item_advanced' | translate}}"
+                data-description="{{:: 'pci_notebook_add_resources_usecase_item_advanced_description' | translate}}"
+                data-model="$ctrl.notebook.selected.resource.usage"
+                data-values="[$ctrl.NOTEBOOK_RESOURCES.ADVANCED]"
+                data-on-change="$ctrl.onUsecaseChange($ctrl.NOTEBOOK_RESOURCES.ADVANCED_FLAVOR)"
+            >
+            </oui-select-picker>
+        </div>
+    </div>
+
+    <!--Notebook resources flavors to choose-->
+    <div class="mb-4">
+        <!--Resource title-->
+        <p class="oui-heading_5 mb-2">
+            <strong
+                data-translate="pci_notebook_add_resources_proposed_title"
+            ></strong>
+        </p>
+
+        <!--Description-->
+        <p data-translate="pci_notebook_add_resources_proposed_description"></p>
+
+        <!--Notebook flavors-->
+        <div class="row">
+            <oui-select-picker
+                class="d-inline-block col-md-6 col-xl-3 my-3"
+                data-ng-repeat="flavor in $ctrl.flavors | filter: { type: $ctrl.notebook.selected.resource.flavorType } track by flavor.name"
+                data-name="{{:: flavor.name }}"
+                data-description="{{:: flavor.description}}"
+                data-model="$ctrl.notebook.selected.resource.flavor"
+                data-values="[flavor]"
+                data-variant="light"
+                data-on-change="$ctrl.onFlavorChange(modelValue)"
+                data-required
+            >
+                <oui-select-picker-label>
+                    <strong data-ng-bind=":: flavor.name | uppercase"></strong>
+                </oui-select-picker-label>
+
+                <oui-select-picker-section>
+                    <div class="d-flex">
+                        <span
+                            data-translate="pci_project_flavors_spec_ram"
```

Comment:
I think we will have to load these translations from the flavors-list component...

Reply:
True, fixed.